### ISSUE TITLE
Ansible installer fails on GCP centos7 image

### ISSUE THREAD

**Issue by Kushmaro** [elastic/ansible-elastic-cloud-enterprise#51]
## Issue Description
The ansible-playbook fails with the following error when run on centos7 using this image (standard centos in gcp)
image - projects/centos-cloud/global/images/centos-7-v20191210

## Error message -
Ece version: 2.4.3
```
TASK [ansible-elastic-cloud-enterprise : include_tasks] ************************
included: /Users/omerkushmaro/.ansible/roles/ansible-elastic-cloud-enterprise/tasks/bootstrap/primary/install_stack.yml for 35.184.83.18
TASK [ansible-elastic-cloud-enterprise : Execute the primary installation] *****
fatal: [35.184.83.18]: FAILED! => {"changed": true, "cmd": "/home/elastic/elastic-cloud-enterprise.sh install --availability-zone us-central1-a --cloud-enterprise-version 2.4.3 --docker-registry docker.elastic.co --ece-docker-repository cloud-enterprise --memory-settings ' {\"runner\":{\"xms\":\"1G\",\"xmx\":\"1G\"},\"proxy\":{\"xms\":\"8G\",\"xmx\":\"8G\"},\"zookeeper\":{\"xms\":\"4G\",\"xmx\":\"4G\"},\"director\":{\"xms\":\"1G\",\"xmx\":\"1G\"},\"constructor\":{\"xms\":\"4G\",\"xmx\":\"4G\"},\"admin-console\":{\"xms\":\"4G\",\"xmx\":\"4G\"}}'", "delta": "0:00:00.520743", "end": "2020-01-30 17:16:30.355823", "msg": "non-zero return code", "rc": 1, "start": "2020-01-30 17:16:29.835080", "stderr": "", "stderr_lines": [], "stdout": "\u001b[0;31mCan't determine a default HOST_IP ('ip' tool can't be found). Please supply '--host-ip' with the appropriate ip address.\u001b[0m", "stdout_lines": ["\u001b[0;31mCan't determine a default HOST_IP ('ip' tool can't be found). Please supply '--host-ip' with the appropriate ip address.\u001b[0m"]}
```

## To Recreate
use [this](https://github.com/elastic/examples/tree/master/Cloud%20Enterprise/Getting%20Started%20Examples/gcp/terraform) terraform example for installing 3 node ece instance in gcp
- needs terraform 0.12 installed 
- need to change the image id in servers.tf from ubuntu, to the image mentioned in the description above, and run the thing :)

**Comment by vaubarth** [comment]
So this seems to be a bit different for various images, but we should simply install the appropriate package to be sure.

**Comment by obierlaire** [comment]
I'm working on creating an AMI with this playbook, and I'm facing the same issue with our internal terraform script using my newly created AMI:

```
TASK [. : Execute the primary installation] ************************************

fatal: [ec2-54-82-208-28.compute-1.amazonaws.com]: FAILED! => {"changed": true, "cmd": "/home/elastic/elastic-cloud-enterprise.sh install --availability-zone us-east-1e --cloud-enterprise-version 2.6.0-SNAPSHOT --docker-registry docker.elastic.co --ece-docker-repository cloud-ci --memory-settings ' {\"runner\":{\"xms\":\"1G\",\"xmx\":\"1G\"},\"proxy\":{\"xms\":\"8G\",\"xmx\":\"8G\"},\"zookeeper\":{\"xms\":\"4G\",\"xmx\":\"4G\"},\"director\":{\"xms\":\"1G\",\"xmx\":\"1G\"},\"constructor\":{\"xms\":\"4G\",\"xmx\":\"4G\"},\"admin-console\":{\"xms\":\"4G\",\"xmx\":\"4G\"}}'", "delta": "0:00:00.325280", "end": "2020-03-10 18:35:06.481538", "msg": "non-zero return code", "rc": 1, "start": "2020-03-10 18:35:06.156258", "stderr": "", "stderr_lines": [], "stdout": "\u001b[0;31mCan't determine a default HOST_IP ('ip' tool can't be found). Please supply '--host-ip' with the appropriate ip address.\u001b[0m", "stdout_lines": ["\u001b[0;31mCan't determine a default HOST_IP ('ip' tool can't be found). Please supply '--host-ip' with the appropriate ip address.\u001b[0m"]}
```

(`Can't determine a default HOST_IP ('ip' tool can't be found)...`). This is unexpected and odd:

- This ip tool is well installed and well usable and running before this ansible step is run (I added an extra step in ansible playbook to check that)
- If I launch an instance with this same AMI manually, I can install ECE with the same bootstrapper with the same params
- If I ssh into the instance created by our CI script and run the ECE installer, everything is fine, even though the ansible logs show that this step was failing!

**Comment by obierlaire** [comment]
Ok, found the fix for Centos AWS AMI: 
https://github.com/elastic/ansible-elastic-cloud-enterprise/pull/55/commits/a8df0d5cbef0646d0fdea15c43048257b70feeec

**Comment by vaubarth** [comment]
This should be fixed by #55
